.travis.yml

```diff
@@ -105,10 +105,10 @@ install:
 
 script:
   - if [[ $TEST_TARGET == 'default' ]]; then
-      python -m iris.tests.runner --default-tests --system-tests --print-failed-images;
+      python -m iris.tests.runner --default-tests --system-tests;
```

Comment:
for now i'm putting this back in; if it is removed then we can discuss removing the functionality too